onadata/libs/templates/account_lockout/lockout_start.txt

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+{% blocktrans with lockout_time=lockout_time username=username %}
+Hi {{ username }},
+
+You have been locked out from your Ona account because of too many wrong passwords.
+Please wait for {{ lockout_time }} minutes before trying to access your account again.
+If this was not you please reach out to our support team for help.
```

Comment:
This should be, "If this was not you, please reach out to our support team for help"

Also, do we provide them with how to access our support team - support@ona.io - do we need to provide the email or can we just use the in-app chat?